src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java

```diff
@@ -160,16 +183,28 @@ public Report(long time, String type, List<Observation> observations) {
     public Claim claim;
     public boolean administration;
 
+    /**
+     * Constructor for Medication HealthRecord Entry.
+     */
     public Medication(long time, String type) {
       super(time, type);
       this.reasons = new ArrayList<Code>();
-      this.claim = new Claim(this);
+      if (Boolean.parseBoolean(Config.get("generate.health_insurance", "false"))) {
+        // Create a medication claim.
+        this.claim = new Claim(this, person);
```

Comment:
Always do this, insurance should not be something you can turn off.

Reply:
I add the ability to enable/disable health insurance because I wasn’t sure if users may want health records where everyone gets the care they need as opposed to sometimes not receiving care if the payer doesn’t cover it. It has a very minimal effect on the the length of the simulation, ~4%. I will remove that though.